packages/components/src/logos/big-sky-logo/index.tsx

```diff
@@ -1,49 +1,72 @@
-interface BigSkyLogoProps extends Omit< React.SVGProps< SVGSVGElement >, 'height' | 'width' > {
-	height?: number;
-	monochrome?: boolean;
-	title?: string;
-	normalized?: boolean;
-}
-
-const Mark = ( {
-	height = 24,
-	monochrome = false,
-	title = 'Big Sky logo',
-	normalized = false,
-	...props
-}: BigSkyLogoProps ) => {
-	const scale = normalized ? 0.8 : 1;
-	const width = height;
-	const color = monochrome ? 'currentColor' : '#030FB0';
+import { forwardRef, useId } from 'react';
+import { CommonLogoProps } from '../types';
+
+type BigSkyLogoProps = CommonLogoProps & Omit< React.SVGProps< SVGSVGElement >, 'width' >;
+
+const CentralLogo = ( { fill }: Pick< BigSkyLogoProps, 'fill' > ) => {
+	return (
+		<>
+			<path
+				fill={ fill }
+				d="m19.223 11.55-3.095-1.068a4.21 4.21 0 0 1-2.61-2.61L12.45 4.777c-.145-.426-.755-.426-.9 0l-1.068 3.095a4.21 4.21 0 0 1-2.61 2.61L4.777 11.55c-.426.145-.426.755 0 .9l3.095 1.068a4.21 4.21 0 0 1 2.61 2.61l1.068 3.095c.145.426.755.426.9 0l1.068-3.095a4.21 4.21 0 0 1 2.61-2.61l3.095-1.068c.426-.145.426-.755 0-.9Zm-3.613.68-1.547.533a2.105 2.105 0 0 0-1.306 1.305l-.533 1.548a.24.24 0 0 1-.453 0l-.534-1.548a2.105 2.105 0 0 0-1.305-1.305l-1.548-.534a.24.24 0 0 1 0-.453l1.548-.534a2.105 2.105 0 0 0 1.305-1.305l.534-1.547a.24.24 0 0 1 .453 0l.534 1.547c.21.615.695 1.095 1.305 1.305l1.547.534a.24.24 0 0 1 0 .453Z"
+			/>
+			<path
+				fill={ fill }
+				d="m13.247 12.079-.535.182a.734.734 0 0 0-.45.45l-.184.536c-.026.072-.13.072-.156 0l-.183-.535a.734.734 0 0 0-.45-.45l-.535-.184c-.072-.026-.072-.13 0-.156l.535-.183a.734.734 0 0 0 .45-.45l.183-.535c.026-.072.13-.072.156 0l.183.535a.734.734 0 0 0 .45.45l.536.183c.072.026.072.13 0 .156Z"
+			/>
+		</>
+	);
+};
+
+export const Mark = forwardRef< SVGSVGElement, BigSkyLogoProps >( function Mark(
+	{ height = 24, monochrome = false, title = 'Big Sky', ...props },
+	ref
+) {
+	const titleId = useId();
+	const maskId = useId();
+
+	const brandFill = monochrome ? 'currentColor' : '#030FB0';
+	const isMasked = monochrome;
+
+	const trimmedTitle = title.trim();
```

Comment:
Why do we `trim()` ? I feel like we should expect consumers to pass a well-formed string. And if this is part of the truthiness checks, I'd rather we handle this as a properly-empty nullish value rather than an empty string.

Also, is the idea with a falsey `title` that it'd be used as a decorative image? And if so, would we also need to hide it from assistive technology (e.g. `aria-hidden`, `role=presentation`) ? Or should we just not support the falsey title altogether?

Reply:
Good points.

- I removed the `trim()` call, but kept `title` as a pure string
- I added more explicit documentation about using `aria-hidden` as the recommended way to remove from assistive technology

This approach follows the specs that from DS-216  (which originated from [this conversation](https://github.com/Automattic/wp-calypso/pull/103402#discussion_r2093088872))